src/constant.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+import {cast} from './cast.js';
+import {Tensor, sizeOfShape} from '../src/lib/tensor.js';
+
+/**
+ * Create a constant array of specified data type and shape,
+ * which contains data incrementing by step.
+ * @param {Array} outputShape
+ * @return {Tensor}
+ */
+
+export function constant(start, step, outputShape, type = 'float32') {
+  const outputElementCount = sizeOfShape(outputShape);
+  const resultArray = [];
+  for (let i = 0; i < outputElementCount; i++) {
+    resultArray.push(start + i * step);
+  }
+  const resultToTensor = new Tensor([resultArray.length], resultArray);
+  const transformTensorType = cast(resultToTensor, type);
```

Comment:
Since we did not make corresponding checks or specifications for data out of bounds in the #63 cast function, should we clarify the behavior of the cases here?
1. constant(start = 127, step = 1,  outputShape = [3], type = 'int8');
2. constant(start = -10, step = 1, outputShape = [3, 3], type = 'uint32');
3. ...

Reply:
Different hardware will behave differently (CPU, NPU, GPU), so I think these cases may be unnecessary.